services/ui/lib/railway-database.ts

```
@@ -32,8 +32,7 @@ class RailwayDatabase {
    */
   private getPoolClass(): new (config: object) => Pool {
     // Dynamic import for server-side only to prevent browser bundling
-    // eslint-disable-next-line @typescript-eslint/no-require-imports
-    const pg = require('pg');
+    const pg = eval('require')('pg');
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)